File: food-app/src/app/models/food.ts
export class Food{
  
   constructor(
               public name:string,
               public composition?:string,
               public photoName?:string,
               public photoBase64?:string,
               public categoryId?:number,
               public subcategoryId?:number,
               public id:number=0,
               public price:number=0
              ){}
}

File: FoodService/DAL/Context/Food.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Context
{
    public class Food
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public double Price { get; set; }
        public string? Composition { get; set; }
        public string? PhotoName { get; set; }
        public string? PhotoBase64 { get; set; }
        public int? CategoryId { get; set; }
        public virtual Category? Category { get; set; }
        public int? SubcategoryId { get; set; }
        public virtual Subcategory? Subcategory { get; set; }

    }
}
